src/Microsoft.IIS.WebManager/gulpfile.js

```diff
@@ -0,0 +1,128 @@
+"use strict";

```

Comment:
Our project would now build using this gulpfile which is standard for WAC extensions